Resources/Prototypes/_Floof/Entities/Mobs/NPCs/pets.yml

```
@@ -18,3 +18,58 @@
     tags:
     - CannotSuicide
     - VimPilot
+
+- type: entity
+  name: Pet Based
+  parent: [MobGiantSpider, BaseItem]
+  id: MobPetBase
+  description: Rescue by the SESWC. SESWC vowed to Heal, raise, and rehome as much as possible.
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Bloodstream
+    bloodMaxVolume: 60 #same as Hamster
+    bloodReagent: CopperBlood #bc spider
+  - type: MobThresholds #same as Mouse
+    thresholds:
+      0: Alive
+      10: Critical
+      20: Dead
+  - type: Item
+    size: Tiny
+  - type: Sprite  # Used some of the Unused sprites  this is Midwife from Tg Station
+    drawdepth: SmallMobs
+    scale: 0.5, 0.5
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base", "movement"]
+      state: midwife
+      sprite: Mobs/Animals/spider.rsi
+  - type: SpriteMovement
+    movementLayers:
+      movement:
+        state: midwife-moving
+    noMovementLayers:
+      movement:
+        state: midwife
+  - type: DamageStateVisuals
+    states:
+      Alive:
+        Base: midwife
+      Dead:
+        Base: midwife_dead
+
+- type: entity
+  name: Pet spider
+  parent: MobPetBase
+  id: MobPetSpider
+  description: Rescue by the SESWC. SESWC vowed to Heal, raise, and rehome as much as possible
```

Cleanup: name and description are inherited from the parent; only need to update the parent field to match the previous suggestion
```suggestion
  parent: MobPetSpiderBase
  id: MobPetSpider
```